### PR TITLE
chore(transformer): turns off async_to_generator plugin in `enable_all`

### PR DESCRIPTION
**Pull request by Dunqing**
The plugin is not ready to test

### CODE REVIEW

**Comment by Dunqing** [comment]
* **#6554** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6554?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Dunqing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/oxc_transformer/src/options/transformer.rs [hunk 1]
```diff
@@ -89,7 +89,10 @@ impl TransformOptions {
             },
             es2016: ES2016Options { exponentiation_operator: true },
             es2018: ES2018Options { object_rest_spread: Some(ObjectRestSpreadOptions::default()) },
-            es2017: ES2017Options { async_to_generator: true },
+            es2017: ES2017Options {
+                // Turned off because it is not ready.
+                async_to_generator: true,
```
Comment (overlookmotel) [inline crates/oxc_transformer/src/options/transformer.rs:94]: Shouldn't this be `false`?

```suggestion
                async_to_generator: false,
```

File: tasks/benchmark/benches/transformer.rs [hunk 2]
```diff
@@ -39,6 +39,7 @@ fn bench_transformer(criterion: &mut Criterion) {
                 // `enable_all` enables all transforms except arrow functions transform
```
Comment (overlookmotel) [inline tasks/benchmark/benches/transformer.rs:39]:
```suggestion
                // `enable_all` enables all transforms except arrow functions transform
                // and async-to-generator
```

Comment (Dunqing) [reply tasks/benchmark/benches/transformer.rs:39]: The comment sounds incorrect, shouldn't include `arrow-functions` and `async-to-generator`?

Comment (Dunqing) [reply tasks/benchmark/benches/transformer.rs:39]: Now I understand

**Comment by Dunqing** [comment]
### Merge activity

* **Oct 14, 9:23 PM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

**Pull request merged**